volontulo/tests/views/test_offers.py

```diff
@@ -137,3 +137,96 @@ def test_offers_list_for_admin(self):
         self.assertIn('offers', response.context)
         # pylint: disable=no-member
         self.assertEqual(len(response.context['offers']), 2)
+
+
+class TestOffersCreate(TestCase):
+    u"""Class responsible for testing offer's create page."""
+
+    @classmethod
+    def setUpTestData(cls):
+        u"""Set up data for all tests."""
+        organization = Organization.objects.create(
```

Comment:
Maybe we should, but I doubt it will right now work properly and we haven't limit any functionality using permisions.

It will be added in near future.